The path to hal/fdi is hardcoded in the source. Foresight has its hal/fdi in
/usr/share. I don't know which is "correct" so for the moment patch it to work
with our hal.

diff -r 48e2a4ad2fe5 src/main.c
--- a/src/main.c	Sat Apr 07 00:08:21 2007 +0100
+++ b/src/main.c	Sat Apr 07 00:13:56 2007 +0100
@@ -32,9 +32,9 @@
 #include <unistd.h>
 
 
-const char      *hal_conf_dir = "/etc/hal/fdi/policy/";
-const char      *write_policy = "/etc/hal/fdi/policy/20-ntfs-config-write-policy.fdi";
-const char      *ro_policy = "/etc/hal/fdi/policy/20-ntfs-config-ro-policy.fdi";
+const char      *hal_conf_dir = "/usr/share/hal/fdi/policy/";
+const char      *write_policy = "/usr/share/hal/fdi/policy/20-ntfs-config-write-policy.fdi";
+const char      *ro_policy = "/usr/share/hal/fdi/policy/20-ntfs-config-ro-policy.fdi";
 const char      *fstab_sav = "/etc/fstab.pre-ntfs-config";
 
 const gchar     fstab_header[] = "# /etc/fstab: static file system information.\n"
